### ISSUE TITLE
use keras tf.nn.avg_pool-dnn PoolForward launch failed

### ISSUE THREAD

**Issue by pass-lin** [ROCmSoftwarePlatform/tensorflow-upstream#1332]
This is code
q_=tf.nn.avg_pool(q,ksize=[1,1,self.pool_size,1],strides=[1,1,self.pool_size,1],
                       padding='SAME',data_format='NHWC',name='q_pool')
k_=tf.nn.avg_pool(k,ksize=[1,1,self.pool_size,1],strides=[1,1,self.pool_size,1],
                       padding='SAME',data_format='NHWC',name='k_pool')

q and k is tensor by Multi-headed attention,it's shape is [batch_size,n_head,squence_length,head_dim]

and error is
dnn PoolForward launch failed
	 [[node ny_encode_layer_1/nyattention_1/q_pool (defined at D:\python\lib\site-packages\keras\backend\tensorflow_backend.py:3009) ]] [Op:__inference_keras_scratch_graph_12280]

Function call stack:
keras_scratch_graph

I use keras.layers.average2D replace it also have same error

**Comment by sunway513** [comment]
Hi @pass-lin , can you fill in the information in the github issue template? 
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/issues/new/choose

**Comment by ppanchad-amd** [comment]
Hi @pass-lin. Apologies for the lack of response. Do you still need assistance with this ticket? Thanks!

**Comment by pass-lin** [comment]
> Hi [@pass-lin](https://github.com/pass-lin). Apologies for the lack of response. Do you still need assistance with this ticket? Thanks!

No, I've already given up on TensorFlow. Keras3+JAX doesn't have this problem.
